### ISSUE TITLE
Keyboard accessibility

### ISSUE THREAD

**Issue by joedolson** [helen/wp-media-grid-view#13]
Most of the controls in the view are not keyboard accessible because they don't use natively focusable elements: Small/Medium/Large toggles and the media grid icons/images themselves should be links or buttons, so that native events are fired on :focus and activation.

Since none of these controls are keyboard focusable, I wasn't able to test the results of activating them from the keyboard, but what should happen is that when an image icon is activated, keyboard focus is moved to the interaction panel to edit that image.

**Comment by ericandrewlewis** [comment]
Hey Joe,

thanks for reviewing this! However, it's probably not a good idea to add proper accessibility to the master or persistent-sidebar branches, as they are more for prototyping UI rather than what will be the finished product.

We'll be incorporating into existing Media JS, so the canonical ticket for this issue where a lot of work has already gone in is [23560 on WP trac](https://core.trac.wordpress.org/ticket/23560). I'm going to close this out, as it's not exactly in scope of our work in this silo.

**Comment by joedolson** [comment]
Sounds good. Thanks for following up.
